build.ps1

```diff
@@ -46,12 +46,12 @@ foreach($packageEntry in $projectFile.Project.Packages.Package) {
     Write-Host "Copying source files..."
     foreach($assetGroup in $packageDef.AssetGroups.AssetGroup) {
         $dest = Join-Path $packagePath $assetGroup.OutputDir
-        foreach($assetSubFolder in Get-ChildItem -Path $assetGroup.AssetDir -Attributes Directory) {
+        foreach($assetSubFolder in Get-ChildItem -Path $assetGroup.AssetDir) {
```

Comment:
The reason `Directories` was here was to avoid problems below with the recursive copy on non-folders.

Reply:
Not quite sure I follow.   That's the only recursive copy in this from what I can see.  Below, you recurse to look for files to put in the manifest, but you don't do any copying.

Reply:
for the permanent record, discussed in channel and we're good with this.